packages/aggregator/src/analytics/aggregateAnalytics/adjustOptionsToAggregationList.js

```diff
@@ -116,7 +116,9 @@ const getAdjustedOrganisationUnitsAndAggregations = async (
 export const adjustOptionsToAggregationList = async (context, fetchOptions, aggregationOptions) => {
   const { aggregations: aggregationList = [] } = aggregationOptions;
   if (aggregationList.length === 0) {
-    return [fetchOptions, aggregationOptions];
+    // Trim off any pre-existing aggregations from fetch options
+    const { aggregations, ...restOfFetchOptions } = fetchOptions;
+    return [restOfFetchOptions, aggregationOptions];
```

Comment:
Bit of a hacky fix here, the issue that was occurring is because we pass through the fetch options to the IndicatorApi which in turn passes them through to each required dataFetch/parameterFetch. Since we add the aggregation options list earlier in the chain they persist here even if no aggregations are requested.

I think the ideal fix here would be for each consumer of the fetch options to rebuild the options containing only the fields that make sense to them (ie. the IndicatorApi itself would end up dropping the aggregations list from the fetch options).

However I think there's still room for brainstorming on the best way to do that, and this is an acceptable solution imo